boards.txt

```diff
@@ -228,7 +228,11 @@ Generic_nRF51822.build.board=GENERIC
 Generic_nRF51822.build.core=nRF5
 Generic_nRF51822.build.variant=Generic
 Generic_nRF51822.build.variant_system_lib=
+<<<<<<< Updated upstream
+Generic_nRF51822.build.extra_flags=-DNRF51 -DNRF51822
+=======
 Generic_nRF51822.build.extra_flags=-DNRF51
+>>>>>>> Stashed changes
```

Comment:
Take a look at the picture posted above and my discussion with @sandeepmistry, the board from Nordic has just pins `0, 1, 3, 5, 9, 11, 20, 21, 24` broken out. for a better user experience I think it's good to maintain pin 24 as `digitalRead(24)` for example.. Should I populate the other numbers on the array, with `(uint32_t)-1`, or their real number, even though they won''t be used?